EIPS/eip-7982.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: 7982
```

Comment:
```suggestion
eip: 7990
```
pls don't self assign the eip number, this is the number assigned to this EIP, rename file as well as the disccussion to url accordingly